RiotSwiftUI/Modules/UserSessions/UserSessionOverview/UserSessionOverviewModels.swift

```diff
@@ -0,0 +1,55 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+// MARK: - Coordinator
+
+enum UserSessionOverviewCoordinatorResult {
+    case openSessionDetails(session: UserSessionInfo)
+}
+
+// MARK: View model
+
+enum UserSessionOverviewViewModelResult: Equatable {
+    case showSessionDetails(sessionInfo: UserSessionInfo)
+    case verifyCurrentSession
+}
+
+extension UserSessionOverviewViewModelResult {
+    static func == (lhs: UserSessionOverviewViewModelResult, rhs: UserSessionOverviewViewModelResult) -> Bool {
```

Comment:
This is not used, let's get rid of the Equatable conformance

Reply:
This is needed for `UserSessionOverviewViewModelTests`

Reply:
Oh okay, let's then add Equatable to UserSessionInfo (automatic) and get rid of the manual implementation here.

Reply:
But I would still need to implement `Equatable` function for `UserSessionOverviewViewModelResult`. So it would look something like this:

```
extension UserSessionOverviewViewModelResult {
    static func == (lhs: UserSessionOverviewViewModelResult, rhs: UserSessionOverviewViewModelResult) -> Bool {
        switch (lhs, rhs) {
        case (.verifyCurrentSession, .verifyCurrentSession):
            return true
        case (let .showSessionDetails(session), let .showSessionDetails(session2)):
            return session == session2
        default:
            return false
        }
    }
}
```
Maybe better would be to add this Equatable extension just to `UserSessionOverviewViewModelTests`? Because for the test purpose we can consider that two sessions are equal if ids are the same, but for other purposes, maybe we need to compare all properties of the session